libvast/vast/format/simdjson.hpp

```diff
@@ -0,0 +1,302 @@
+/******************************************************************************
+ *                    _   _____   __________                                  *
+ *                   | | / / _ | / __/_  __/     Visibility                   *
+ *                   | |/ / __ |_\ \  / /          Across                     *
+ *                   |___/_/ |_/___/ /_/       Space and Time                 *
+ *                                                                            *
+ * This file is part of VAST. It is subject to the license terms in the       *
+ * LICENSE file found in the top-level directory of this distribution and at  *
+ * http://vast.io/license. No part of VAST, including this file, may be       *
+ * copied, modified, propagated, or distributed except according to the terms *
+ * contained in the LICENSE file.                                             *
+ ******************************************************************************/
+
+#pragma once
+
+#include "vast/concept/hashable/hash_append.hpp"
+#include "vast/concept/hashable/xxhash.hpp"
+// #include "vast/concept/parseable/vast/json.hpp" // TO DELETE?
+#include "vast/fwd.hpp"
+
+#include "vast/defaults.hpp"
+#include "vast/detail/flat_map.hpp"
+#include "vast/detail/line_range.hpp"
+#include "vast/detail/string.hpp"
+#include "vast/error.hpp"
+#include "vast/format/multi_layout_reader.hpp"
+#include "vast/format/ostream_writer.hpp"
+#include "vast/json.hpp" // TO DELETE?
+#include "vast/logger.hpp"
+#include "vast/schema.hpp"
+#include "vast/view.hpp" // Added
+
+#include <caf/expected.hpp>
+#include <caf/fwd.hpp>
+#include <caf/settings.hpp>
+
+#include <chrono>
+#include <simdjson.h>
+
+namespace vast::format::simdjson {
+
+// class writer : public ostream_writer {
+// public:
+//   using defaults = vast::defaults::export_::json;
+
+//   using super = ostream_writer;
+
+//   using super::super;
+
+//   caf::error write(const table_slice& x) override;
+
+//   const char* name() const override;
+// };
```

Comment:
This you can just remove, as simdjson only concerns the reader.

Reply:
Ok